example/bump-chart.md

```diff
@@ -0,0 +1,29 @@
+# Bump Chart
+
+When constructing data to be used with the [BumpChart](http://d3plus.org/docs/#BumpChart) class, the data provided for the y-axis must be an integer rank. Given this data array:
```

Comment:
I know this may sound silly/vague, but can you spruce up this text to include a sentence or two on *why* someone would use a bump chart? Maybe reference rankings over time?